### DISCUSSION TITLE
(/usr/lib/libharfbuzz.so.0: undefined symbol: FT_Get_Color_Glyph_Layer)

### DISCUSSION THREAD

**Discussion by 3xploiton3** [harfbuzz/harfbuzz#4609] [Q&A] [answered]
`/opt/lampp/bin/php -m`

> PHP Warning: PHP Startup: Unable to load dynamic library 'imagick.so' (tried: /opt/lampp/lib/php/extensions/no-debug-non-zts-20220829/imagick.so (/usr/lib/libharfbuzz.so.0: undefined symbol: FT_Get_Color_Glyph_Layer), /opt/lampp/lib/php/extensions/no-debug-non-zts-20220829/imagick.so.so (/opt/lampp/lib/php/extensions/no-debug-non-zts-20220829/imagick.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0

reference

https://community.apachefriends.org/f/viewtopic.php?f=17&t=82021#p281544

OS: Arch Linux x86_64

**Comment by behdad** [comment, marked as answer]
This means that an older freetype.so is being linked to at runtime...
